analyzer/analyzer.c

```diff
@@ -1241,7 +1243,16 @@ extern bool printFields(const Pgn *pgn, const uint8_t *data, int length, bool sh
       break;
     }
 
-    strcpy(fieldName, field->camelName ? field->camelName : field->name);
+    if ( showCamel )
```

Comment:
`if (showCamel)` is consistent with existing code.

Reply:
there is a .clang-format file in the repo, just run `make format` with clang-format installed to fix whitespace and other formatting.